### PR TITLE
Use the amqp_ prefix for AMQP variables

### PR DESCRIPTION
**Pull request by elemoine**

### CODE REVIEW

**Review by simonpasquier — APPROVED** [review]

**Review by ityaptin — APPROVED** [review]

File: heka/meta/heka.yml [hunk 1]
```diff
@@ -476,6 +476,7 @@ aggregator:
 {%- endif %}
 ceilometer_collector:
   decoder:
```
Comment (SwannCroiset) [inline heka/meta/heka.yml:478]: decoder: will null if `ceilometer_collector.amqp_host` is not defined
which is bad

File: heka/meta/heka.yml [hunk 2]
```diff
@@ -484,21 +485,22 @@ ceilometer_collector:
         decoder: 'ceilometer'
         decode_resources: {{ ceilometer_collector.resource_decoding|lower }}
         metadata_fields: "status deleted container_format min_ram updated_at min_disk is_public size checksum created_at disk_format protected instance_host host display_name instance_id instance_type status state user_metadata.stack"
+{%- endif %}
   input:
```
Comment (SwannCroiset) [inline heka/meta/heka.yml:489]: ditto

File: heka/meta/heka.yml (hunk 1 above)
Comment (SwannCroiset) [reply heka/meta/heka.yml:478]: but it is not the goal of this patch ..

**Review by SwannCroiset — APPROVED** [review]

File: heka/meta/heka.yml (hunk 1 above)
Comment (elemoine) [reply heka/meta/heka.yml:478]: Yes, that should be handled separately.

**Review by ityaptin — APPROVED** [review]

**Pull request merged**